test/jeison-test.el

```diff
@@ -234,11 +234,14 @@
                                   }"
                           '(a (jeison:filter-candidates b) number))))))
 
-(ert-deftest jeison:check-hash-table-of-invalid-key-type ()
+(ert-deftest jeison:check-cl-deftype-jeison-hash-table-of ()
```

Comment:
This test is incorrect, it doesn't test Jeison's type checking functionality. Refer to my previous comments for explanation.